packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -105,16 +115,49 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
     insertIndex = -1;
   };
 
+  const _onDragOverAutofill = (event?: React.DragEvent<HTMLDivElement>) => {
+    if (autofillDragDropEnabled) {
+      event?.preventDefault();
+    }
+  };
+
+  const _onDropAutoFill = (event?: React.DragEvent<HTMLDivElement>) => {
+    if (props.onDropAutoFill) {
+      props.onDropAutoFill(event);
+    } else {
+      insertIndex = selectedItems.length;
+      event?.preventDefault();
```

Comment:
do we want to do event?.preventDefault irrespective ? Since we would be leaving that responsibility on props.onDropAutoFill otherwise.

Reply:
I think that's ok? If they choose to override it, they can control this? There are times you may want to call preventDefault selectively

Reply:
from what i have seen, we usually do end up doing this, since it overrides expected behavior and ends up doing unwanted interactions.. maybe worth while to do so irrespective , than leavng it on the props, to figure it out/